cads_adaptors/adaptors/cadsobs/adaptor.py

```diff
@@ -7,6 +7,23 @@
 
 logger = logging.getLogger(__name__)
 
+possible_aux_fields = [
```

Comment:
What is this used for, I don't see it referenced elsewhere in the code

Reply:
You are right, I removed it.